### DISCUSSION TITLE
I cannot pass the education verification over four times, I am a Chinese Student from YIT

### DISCUSSION THREAD

**Discussion by Tang-XC** [community/community#51447] [GitHub Education]
how can i fix this problem

**Comment by samus-aran** [comment]
Hi @Tang-XC  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by Tang-XC** [comment]
I tried many times,but always send me this information back,the proof I provided is completely accurate.Is it a computer audit errorI have attached my school status verification report of CHSI and my&nbsp;Student ID card.Can you help me with the review.





&nbsp;


油炸知了猴
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Ashley ***@***.***&gt;; 
发送时间: 2023年3月31日(星期五) 凌晨5:02
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [community/community] I cannot pass the education verification over four times, I am a Chinese Student from YIT (Discussion #51447)





 
Hi @Tang-XC  👋
 We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by whitewatercn** [comment]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack
